common/determined_common/experimental/experiment.py

```diff
@@ -26,20 +26,19 @@ def top_checkpoint(
         smaller_is_better: Optional[bool] = None,
     ) -> checkpoint.Checkpoint:
         """
-        Return the :class:`~determined.experimental.Checkpoint` instance with the best
-        validation metric as defined by the ``sort_by`` and ``smaller_is_better``
+        Return the :class:`~determined.experimental.Checkpoint` for this experiment that
+        has the best validation metric, as defined by the ``sort_by`` and ``smaller_is_better``
```

Comment:
question: would it be helpful to explain how the `validation_metric` is chose if there are multiple of them?

Reply:
There is text to explain this below (`sort_by`), I think.

Reply:
Oh yeah missed that part when reading the doc string.